test/unit/test_jinja.py

```diff
@@ -292,6 +296,15 @@ def test_docs_block_expr(self):
         self.assertEqual(blocks[0].contents, ' asdf {{ "{% enddocs %}" ~ "}}" }}')
         self.assertEqual(blocks[0].block_name, 'more_doc')
 
+    def test_unclosed_model_quotes(self):
+        body = '{% model my_model -%} select * from "something"."something_else{% endmodel %}'
```

Comment:
can you comment in a link to https://github.com/fishtown-analytics/dbt/issues/1533 here and indicate that this test replicates that issue